cmd/wallet-js-sdk/test/specs/didcomm/connect.v2.spec.js

```diff
@@ -0,0 +1,127 @@
+    bob = await loadFrameworks({name: BOB_LABEL_V2, mediaTypeProfiles:["didcomm/v2"], keyType:'ecdsap256ieee1363', keyAgreementType:'p256kw'})
+
+    aliceDID = await createWalletAndPublicDID(alice, ALICE_LABEL_V2, aliceDID)
+    // await createWalletAndPublicDID(bob, BOB_LABEL_V2, bobDID)
+});
+
+export async function createWalletAndPublicDID(agent, agentLabel, agentDID) {
+    try {
+        await createWalletProfile(agent, agentLabel, {localKMSPassphrase: testConfig.walletUserPassphrase})
+
+        let wallet = new UniversalWallet({agent: agent, user: agentLabel})
+        let authResponse = await wallet.open({localKMSPassphrase: testConfig.walletUserPassphrase})
+        expect(authResponse.token).to.not.empty
+
+        let auth = authResponse.token
+        let didManager = new DIDManager({agent: agent, user: agentLabel})
+        let document = await didManager.createOrbDID(auth)
+
+        expect(document).to.not.empty
+
+        agentDID = document.didDocument.id
+
+        expect(agentDID).to.not.empty
+
+    } catch (e) {
+        console.error('failed to create wallet and peer DID document from ${agentLabel} agent ',e)
+        expect.fail(e);
+    }
+
+    return agentDID
+}
+
+after(function () {
+    alice ? alice.destroy() : ''
+    bob ? bob.destroy() : ''
+});
+
+describe('running DIDComm V2 connection tests', async function () {
+    it('alice connected to V2 mediator', async function () {
+        try {
+            await connectToMediator(alice, testConfig.mediatorV2EndPoint, {},{isDIDCommV2: true})
+            let conns = await getMediatorConnections(alice)
+            expect(conns).to.not.empty
+        } catch (e) {
+            console.error('failed to connect alice to mediator ',e)
+            expect.fail(e);
+        }
+    });
+
+    let routerConnections
+    it('bob connected to V2 mediator', async function () {
+        try {
+
+            await connectToMediator(bob, testConfig.mediatorV2EndPoint,{}, {isDIDCommV2: true})
+            let conns = await getMediatorConnections(bob, {single: true})
+            expect(conns).to.not.empty
+
+            routerConnections = [conns]
+        } catch (e) {
+            console.error('failed to connect bob to mediator ',e)
+            expect.fail(e);
+        }
+    });
+
+    let invitation
+    it('alice creates oobv2 invitation for bob', async function () {
+        try {
+            let res = await alice.outofbandv2.createInvitation({
+                label: ALICE_LABEL_V2,
+                body: {accept: ["didcomm/v2"]},
+                from: aliceDID,
+            })
+
+             invitation = res.invitation
+            expect(invitation).to.not.empty
+        } catch (e) {
+            console.error('failed to create invitation from alice agent ',e)
+            expect.fail(e);
+        }
+    });
+
+    it('bob connect to alice by accepting invitation', async function () {
+        try {
+            await wait(5000)
```

Comment:
why waiting for 5 seconds here?? V1 tests doesn't wait.

Reply:
it's needed to allow orb DID creation to be populated in sidetree, otherwise test fails with missing did error